res/values-pt-rBR/strings.xml

```diff
@@ -1364,7 +1364,7 @@ Comandos:
     <string name="Defenders">Defensores</string>
     <string name="Attackers">Atacantes</string>
     <string name="MEGA_SPLIT">MEGA SPLIT 64x</string>
-    <string name="Missing_Permissions">Permissões ausentes</string>
+    <string name="Missing_Permissions">Sem Permissões</string>
```

Comment:
Original text was correct. The text you tried to correct wouldn't make sense if changed like that, because in English it would mean "No Permissions".

Reply:
Entendi, so tentei alterar por conta que Permissões Ausentes não é tão utlizado

Reply:
"Permissões em falta" would sound better in this case then.

Reply:
Ok